src/common/blockchain.sh

```diff
@@ -1,3 +1,201 @@
 #!/usr/bin/env bash
 
 # Common IBM blockchain platform functions, e.g. to provision a blockchain service
+
+# shellcheck disable=2086
+
+# shellcheck source=src/common/utils.sh
+source "${SCRIPT_DIR}/common/utils.sh"
+
```

Comment:
Can we add `set -x` to get beginning of each function  so that we get debug information within the module when it runs?  It is very difficult to follow what is happening without this.